# CapTrap-Seq: A platform-agnostic and quantitative approach for high-fidelity full-length RNA sequencing 
Silvia Carbonell-Sala<sup>1</sup>, Tamara Perteghella1,2, Julien Lagarde1,3, Hiromi Nishiyori4,5, Emilio Palumbo1, Carme Arnan1, Hazuki Takahashi4,5, Piero Carninci4,5, Barbara Uszczynska-Ratajczak1,6,#, Roderic Guigó1,2,#
 
    1. Centre for Genomic Regulation (CRG), the Barcelona Institute of Science and Technology, Barcelona, Catalonia, Spain.
    2. Universitat Pompeu Fabra, Barcelona, Catalonia, Spain
    3. Flomics Biotech, Barcelona, Catalonia, Spain
    4. RIKEN Center for Integrative Medical Sciences (IMS), Yokohama, Japan
    5. Human Technopole, Milan, Italy.
    6. Institute of Bioorganic Chemistry, Polish Academy of Sciences, Poznan, Poland.
# Corresponding author
Correspondence should be addressed to R.G. (roderic.guigo@crg.cat) or B.U.R (buszczynska@ibch.poznan.pl). 

Long-read RNA sequencing is essential to produce accurate and exhaustive annotation of eukaryotic genomes. Despite advancements in throughput and accuracy, achieving reliable end-to-end identification of RNA transcripts remains a challenge for long-read sequencing methods. To address this limitation, we developed CapTrap-seq, a cDNA library preparation method, which combines the Cap-trapping strategy with oligo(dT) priming to detect 5'capped, full-length transcripts. In our study, we evaluated the performance of CapTrap-seq alongside other widely used RNA-seq library preparation protocols in human and mouse tissues, employing both ONT and PacBio sequencing technologies. To explore the quantitative capabilities of CapTrap-seq and its accuracy in reconstructing full-length RNA molecules, we implemented a novel capping strategy for synthetic RNA spike-in sequences that mimics the natural 5’cap formation. Benchmarks produced by us, as well as independent evaluations, demonstrate that CapTrap-seq is a competitive open-source, platform-agnostic RNA library preparation method for producing full-length transcript sequences.

