## ![Figure S4.](FigureS4.pdf) 
# Mapping statistics for CapTrap-seq across different sequencing platforms in human brain and heart. 
**(A)** The number of errors per mapped bases (error rate) for CapTrap-seq for ONT, PacBioSI and PacBioSII sequencing platforms. The color code as specified in Figure S1C; <br> 
**(B)** Proportion of poly(A) (green) and non-poly(A) (red) ONT and PacBio (SI and SII) reads; <br>
**(C)** Support of poly(A) reads by the presence of canonical polyA polyadenylation motif[^45] within a window of +/-50 bp[^36]. <br>
**(D)** Detection of full-length reads in brain and heart samples, across the three sequencing technologies, binned by length. See Figure 1E for details.

[^45]: Lopez, F., Granjeaud, S., Ara, T., Ghattas, B. & Gautheret, D. The disparate nature of ‘intergenic’ polyadenylation sites. Rna 12, 1794–1801 (2006).
[^36]: Uszczynska-Ratajczak, B., Lagarde, J., Frankish, A., Guigó, R. & Johnson, R. Towards a complete map of the human long non-coding RNA transcriptome. Nat. Rev. Genet. 19, 535–548 (2018).
