## Figure 1. 
# Full-length transcript annotation using CapTrap-seq and other library preparation methods. 
**(A)** CapTrap-seq experimental workflow. Gray boxes highlight the four main steps of full-length (FL) cDNA library construction: Anchored dT Poly(A)+, CAP-trapping[^23][^24][^25][^26], CAP and Poly(A) dependent Linker Ligation, and FL-cDNA library enrichment as described in the text. <br>
**(B)** Two adult human complex transcriptomic samples, brain and heart, were used to perform the cross-protocol and cross-platform comparisons to assess the quality of CapTrap-seq. The horizontal green line indicates the cross-protocol comparisons, including four different sequencing library preparation methods: CapTrap-seq, directRNA®, TeloPrime® and SMARTer®. Whereas, the vertical blue line shows cross-platform comparison using CapTrap-seq in combination with three long-read sequencing platforms: ONT, PacBio Sequel I and Sequel II. <br>
**(C)** Read aggregate deptools[^44] profiles along the body of annotated GENCODE genes. <br>
**(D)** Length distribution of mapped long-read ONT reads for each protocol, excluding synthetic RNA spike-in controls. The total number of reads (N), median read length (beige vertical line) and the mapping rate are shown in the top right corner. <br>
**(E)** Detection of full-length reads among all, spliced and unspliced reads with 5′ and 3′ termini inferred from robust (FANTOM5 phase 1 and 2 robust (n = 201,802)) CAGE clusters[^35] and poly(A) tails; Colors highlights four different categories of long-read (LR) completeness: Grey: unsuported LRs; Sky blue: 3’suported LRs; Light pink: 5’ suported LRs; Purple: 5’+3’ supported LRs. The blue percentage displayed at the top of each bar indicates the ratio of a specific read type (spliced, unspliced) to the total number of reads.

[^23]: Carninci, P. et al. High-efficiency full-length cDNA cloning by biotinylated CAP trapper. Genomics 37, 327–336 (1996).
[^24]: Carninci, P. & Hayashizaki, Y. High-efficiency full-length cDNA cloning. Methods Enzymol. 303, 19–44 (1999).
[^25]: Morioka, M. S. et al. Cap Analysis of Gene Expression (CAGE): A Quantitative and Genome-Wide Assay of Transcription Start Sites. Methods Mol. Biol. Clifton NJ 2120, 277–301 (2020).
[^26]: Grapotte, M. et al. Discovery of widespread transcription initiation at microsatellites predictable by sequence-based deep neural network. Nat. Commun. 12, 3297 (2021).
[^35]: Statello, L., Guo, C.-J., Chen, L.-L. & Huarte, M. Gene regulation by long non-coding RNAs and its biological functions. Nat. Rev. Mol. Cell Biol. 22, 96–118 (2021).
[^44]: Ramírez, F. et al. deepTools2: a next generation web server for deep-sequencing data analysis. Nucleic Acids Res. 44, W160–W165 (2016).
